packages/tupaia-web/src/features/Dashboard/Dashboard.tsx

```diff
@@ -181,15 +194,17 @@ export const Dashboard = () => {
               <StaticMap bounds={bounds} />
             )}
           </DashboardImageContainer>
-          <TitleBar>
-            <Title variant="h3">{entity?.name}</Title>
-            {activeDashboard && (
-              <ExportButton startIcon={<GetAppIcon />} onClick={() => setExportModalOpen(true)}>
-                Export
-              </ExportButton>
-            )}
-          </TitleBar>
-          <DashboardMenu activeDashboard={activeDashboard} dashboards={dashboards} />
+          <StickyBar $isExpanded={isExpanded}>
```

Comment:
In the new design the dashboard menu is included in the sticky bar so updating it here